File: app/src/main/java/com/visiontutor/app/TutorDashboard.java
package com.visiontutor.app;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;
import android.view.View;

public class TutorDashboard extends AppCompatActivity implements View.OnClickListener{

    CardView tutProfile;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_tutordashboard);

        tutProfile = findViewById(R.id.tutorprofile);

        tutProfile.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        Intent intent = null;
        switch (v.getId()) {
            case R.id.tutorprofile:
                intent = new Intent(this, TutorProfile.class);break;
        }

        startActivity(intent);
    }
}

File: app/src/main/java/com/visiontutor/app/TutorDashboard.kt
package com.visiontutor.app

import android.content.Intent
import android.os.Bundle
import android.view.View

import androidx.appcompat.app.AppCompatActivity
import androidx.cardview.widget.CardView

class TutorDashboard : AppCompatActivity(), View.OnClickListener {

    private lateinit var tutProfile: CardView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_tutordashboard)

        tutProfile = findViewById(R.id.tutorprofile)

        tutProfile.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        var intent: Intent? = null
        when (v.id) {
            R.id.tutorprofile -> intent = Intent(this, TutorProfile::class.java)
        }

        startActivity(intent)
    }
}
